test/srb-en.html

```diff
@@ -0,0 +1,49 @@
+---
+title: Basic search Page for Governement of Canada using Headless
+description: Demo page for the Canada.ca Search UI Basic
+lang: en
+altLangPage: srb-fr.html
+nositesearch: true
+dateModified: 2024-03-14
+stylesheets:
+    - href: "../src/connector.css"
+---
+
+<!--FORM-->
+<form method="GET" action="#wb-land" class="mrgn-tp-sm">
+	<div class="input-group mrgn-tp-lg">
+	<label class="wb-inv" for="sch-inp-ac">Search Government of Canada websites</label>
+		<input id="sch-inp-ac" class="form-control" value="" name="q" autocomplete="off" spellcheck="false" type="search" data-fusion-query="safe" aria-describedby="gc-pi">
+		<span class="input-group-btn">
+		<button id="sch-inp" class="btn btn-primary btn-small" name="wb-srch-sub" type="submit"> <span class="glyphicon-search glyphicon" aria-hidden="true"></span> <span class="wb-inv">Search</span> </button>
+		</span> </div>
+	<p id="gc-pi" class="mrgn-tp-md">Don’t include personal information (telephone, email, SIN, financial, medical, or work details).</p>
+</form>
+
+<!--RESULTS-->
+<div data-analytics-id="squery">
+	<div class="page-type-search" data-srchprms='{
+			"searchHub": "canada-gouv-public-websites",
+			"organizationId": "employmentandsocialdevelopmentcanadanonproduction14o5d9wry",
+			"accessToken":"xx33fe613d-78eb-45c0-a495-72b54fa7821d",
+			"nbOfSuggestion": 5
+		}'>
+		<section id="query-summary"></section>
+		<section id="result-list" class="results"></section>
+		<div class="text-center">
+			<p class="wb-inv">Search results pages</p>
+			<ul id="pager" class="pagination mrgn-bttm-0"></ul>
+		</div>
+		<p class="text-center small"><a href="https://www.canada.ca/en/sr/srb/sra.html">Perform an advanced search</a></p>
```

Comment:
This `p` for advanced search will be moved outside the div with querySelector = `[data-gc-search]`